src/Network/Network/help/Get-AzApplicationGatewayAvailableServerVariableAndHeader.md

```diff
@@ -0,0 +1,137 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/get-azapplicationgatewayavailableservervariableandheader
+schema: 2.0.0
+---
+
+# Get-AzApplicationGatewayAvailableServerVariableAndHeader
+
+## SYNOPSIS
+Get the supported server variables and available request and response headers.
+
+## SYNTAX
+
+```
+Get-AzApplicationGatewayAvailableServerVariableAndHeader [-ServerVariables] [-RequestHeaders]
+ [-ResponseHeaders] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzApplicationGatewayAvailableServerVariableAndHeader** cmdlet gets the supported server variables and available request and response headers. Parameters can be used to get the variables or headers lists.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\>$availableServerVariables = Get-AzApplicationGatewayAvailableServerVariableAndHeader -ServerVariables
```

Comment:
@nirkum18 it would be nice if these examples weren't set to a variable and we returned the corresponding output for these commands

Reply:
Done